.travis.yml

```diff
@@ -22,8 +25,9 @@ jobs:
         - pip install -c constraints.txt -e .
       script:
         - flake8
-        - black --check print-logs run-query setup.py ethindex
-        - mypy --ignore-missing-imports ethindex
+        - black --check print-logs run-query setup.py ethindex testing
+        - mypy --ignore-missing-imports ethindex testing
+        - py.test testing
```

Comment:
the command should be `pytest` or `python -m pytest`. `py.test` is a leftover from the past.

Reply:
Can we call the folder tests as it is how it usually called and also in our other projects.

Reply:
sure. renamed it.